packages/zudoku/src/lib/plugins/markdown/resolver.ts

```diff
@@ -1,13 +1,8 @@
-import fs from "fs";
```

Comment:
This gave us a warning in every build that these APIs are not available on client side. The method that uses them is not in use (anymore?)

Reply:
great find!!!!